.github/workflows/stale.yml

```
@@ -13,10 +13,23 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v4
+    - name: "Check PR's with 'CLA signed' label"
```

PRs not PR's ([ref](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users))

```suggestion
    - name: "Check PRs with 'CLA signed' label"
```

---

Ah, bad typing habit. Thanks.

---

Oh, this calls for a Life of Brian reference:

https://www.youtube.com/watch?v=DdqXT9k-050